codec_specs.md

```diff
@@ -817,6 +817,16 @@ The `BlockAddId` of the `BlockMore` containing these data **MUST** be 1.
 
 Initialization: none
 
+### A_ATRAC/AT1
+
+Codec ID: A_ATRAC/AT1
+
+Codec Name: Sony ATRAC1 Codec
+
+Description: The original ATRAC codec by Sony, mainly used in MiniDisc platforms.
+
+Initialization: If the encoded audio track has a channel count other than `2` (STEREO), the `Private Data` contains a single unsigned byte that specifies the number of channels. If the channel count is `0` or the `Initialization` is set to `None`, the channel count will default to `2` (STEREO).
```

Comment:
Channel count is already provided by Matrska dedicated element, why is it needed to repeat it in private data part?

Reply:
I agree. If signalling the number of channels is the only thing it's used for, just leave it empty & use the existing element.

Reply:
It seems I misunderstood the purpose of the existing `Channels` attribute. Just to confirm: the Tracks -> TrackEntry -> Audio -> Channels field can be used to express the number of channels encoded in the actual audio data (encoded in a way specific to the used codec), and does not refer to a Matroska-specific way of potentially encoding channels (e.g. using additional clusters/blocks)?

Reply:
I've updated the PR to remove the Initialization as requested.

Reply:
> the Tracks -> TrackEntry -> Audio -> Channels field can be used to express the number of channels encoded in the actual audio data (encoded in a way specific to the used codec), and does not refer to a Matroska-specific way of potentially encoding channels (e.g. using additional clusters/blocks)?

Exactly. And it is mandatory (if not in the file it means that it is 1).

Reply:
Where is the channel information found in the bitstream ? Theoretically we should show say that this information MUST (normative) match the channel field in Matroska. Although I think to save space we should have a general rule in the codec document that states such obvious thing. We still need to know where to find that information in the codec data (if there's one and not relying on a container header).

Reply:
It seems [this library](https://github.com/linux-minidisc/linux-minidisc/blob/43a01190f819e44c5709017b889be9bf8766b989/libhimd/trackindex.c#L134) has more information on all the bitstream (the TypeScript lib you mention seems to borrow from that project). But that's not exactly a documentation.

Reply:
> Where is the channel information found in the bitstream ?

My understanding, because the first version of the PR was containing the channel count in private data, is that this piece of information is not in the bitstream. Not a problem for the spec here, or hat do I miss?

>  Although I think to save space we should have a general rule in the codec document that states such obvious thing.

Well, it is the idea for all duplicated pieces of information, nothing specific here.

Reply:
Sorry, missed this, the information is indeed not present in the bitstream itself. It's something that needs to be encoded in the container.